tgdb/bandetails.php

```diff
@@ -66,6 +66,9 @@
 				case "unbanned_ckey":
 					$value = crossrefify($value,'adminckey');
 					break;
+				case "round_id":
+					$value = crossrefify($value,'round_id');
+					break;
```

Comment:
This isn't needed, in bandetails, columns default to their exact value already, and crossrefify doesn't have a round_id mode, (nor does it need it). crossrefifiy just adds a link to view more details about that thing, (the (b) (c) (pp) links on hover) 

If you remove the changes from this file, and only edit the template, it will work just fine since bandetails exposes every column of the ban to the template, so ROUND_ID already exists just by being in the database.